docs/fragment-arguments.md

```diff
@@ -0,0 +1,31 @@
+# Fragment Arguments
+
+Although Mavericks, doesn't have to be used with Fragments, it has some helpers in case it is.
+
+### Setting Mavericks Fragment arguments
+When creating a Fragment:
+1. Create a `Parcelable` or `Serializable` object
+2. Set the Fragment arguments to `yourArgs.asMavericksArgs()`
+
+```kotlin
+val fragment = YourFragment()
+fragment.arguments = yourArgs.asMavericksArgs()
```

Comment:
I would if we could also/alternatively have an extension `fragment.setMvRxArgs(Parcelable)`